index.html

```diff
@@ -433,6 +443,20 @@ <h2><dfn>disconnect()</dfn> method</h2>
         <a data-cite="!WHATWG-DOM#context-object">context object</a>'s
         <a>observer buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>supportedEntryTypes</dfn> attribute</h2>
+        <p>The user agent must maintain <dfn>supported entry types</dfn>, a list
+        of strings representing the entry types which the user agent supports
+        for the <a>PerformanceObserver</a> interface. This list is populated by
```

Comment:
This list is initially empty and extended by other specifications by calling the ... algorithm?

_p.s. this also means we need to go back and update all the specs that emit such events._

Reply:
Done, and ack